client/src/components/CommentApp/state/comments.ts

```diff
@@ -302,7 +302,7 @@ export const reducer = produce(
       }
       case actions.INVALIDATE_CONTENT_PATH: {
         // Delete any comments that exist in the contentpath
-        const comments = draft.comments;
+        const { comments } = draft;
```

Comment:
Kill this rule with fire, please. :-)

I can't even comprehend why someone would prefer "map this object to an object with one property" over "read this property and set it to a variable", outside of code-golfing challenges. Maybe if there was a way to trigger this rule only when assigning multiple variables, I'd be on board, but it doesn't look like there is.